internal/pkg/config/env_defaults.go

```diff
@@ -50,6 +50,21 @@ const (
 	defaultStatusBurst    = 25
 	defaultStatusMax      = 50
 	defaultStatusMaxBody  = 0
+
+	defaultUploadStartInterval = time.Second * 3
+	defaultUploadStartBurst    = 8
+	defaultUploadStartMax      = 3
+	defaultUploadStartMaxBody  = 1024 * 1024 * 5
```

Comment:
Why so large?

Reply:
this was a shot in the dark --

The file start payloads describe any metadata about the file to be stored. The mapping is dynamic, so any integrations/clients can send what fields they are interested in, which could potentially be large. _This_ large, I have no idea. But I also didn't have a case for necessitating it smaller for the unknown payloads